topdown/parse_bytes.go

```diff
@@ -121,21 +121,34 @@ func extractNumAndUnit(s string) (string, string) {
 	}
 
 	firstNonNumIdx := -1
-	for idx, r := range s {
-		if !isNum(r) {
+	for idx := 0; idx < len(s); idx++ {
+		r := rune(s[idx])
+		if !isNum(r) && r != 'e' && r != 'E' && r != '+' && r != '-' {
```

Comment:
Adding a small note for this check will be helpful.

Reply:
I added a line comment explaining the check!